src/test/java/seedu/address/logic/parser/PrefixTest.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.logic.parser;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import org.junit.jupiter.api.Test;
+
+class PrefixTest {
+
+    private final Prefix name = new Prefix("-n");
+    private final Prefix index = new Prefix("-x");
+
+    @Test
+    public void equalityCheck() {
```

Comment:
Maybe we could rename this to something more specific/detailed? 1 suggestion could be "parse_prefixNotEqual_failure()"

```suggestion
    public void parse_prefixNotEqual_failure() {
```